ipam/store.go

```diff
@@ -176,3 +176,9 @@ func (a *Allocator) deleteFromStore() error {
 	}
 	return store.DeleteObjectAtomic(a)
 }
+
+// DataScope method returns the storage scope of the datastore
+func (a *Allocator) DataScope() datastore.DataScope {
+	//This method is useless currently, hence return a arbitrary value here
+	return datastore.DataScope(-1)
```

Comment:
@aboch with the new ipam extension proposal, I believe the ipam will be used both by local and globally scoped networks and objects. In that case, should we make this configurable in ipam ?

Anyways, that doesnt impact this PR  & it must be Global for now. And can be addressed along with the ipam extension proposal.